plugins/pastefromword/filter/default.js

```diff
@@ -38,7 +38,74 @@
 	 * @private
 	 * @member CKEDITOR.plugins
 	 */
-	CKEDITOR.plugins.pastefromword = {};
+	CKEDITOR.plugins.pastefromword = {
+		/**
+		 * Method parses RTF content to find embedded images. Please be aware that method should only return `png` and `jpeg` images.
+		 *
+		 * @private
+		 * @since 4.8.0
+		 * @member CKEDITOR.plugins.pastefromword
+		 * @param {String} rtfContent RTF content to be checked for images.
+		 * @returns {Object[]} An array of images found in the `rtfContent`.
+		 * @returns {String} return.hex Hexadecimal string of an image embedded in `rtfContent`.
+		 * @returns {String} return.type String represent type of image, allowed values: 'image/png', 'image/jpeg'.
+		 */
+		extractImagesFromRtf: function( rtfContent ) {
```

Comment:
I would like to avoid putting this logic at the very beginning of PFW filter file.

This part is not as important to expose it here. Instead I would suggest enclosing it in a dedicated namespace, like `CKEDITOR.plugins.pastefromword.images` and it could contain:

* `extractImagesFromRtf` actually renamed to `extractFromRtf`.
* `extractImgTagsFromHtml` actually renamed to `extractTagsFromHtml`.

Then move it a bit to the bottom of a file.